### ISSUE TITLE
setup.py cannot import icdiff

### ISSUE THREAD

**Issue by rcfox** [jeffkaufman/icdiff#92]
```
Collecting git+https://github.com/jeffkaufman/icdiff.git
  Cloning https://github.com/jeffkaufman/icdiff.git to /tmp/pip-cCwQlp-build
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-cCwQlp-build/setup.py", line 3, in <module>
        from icdiff import __version__
    ImportError: No module named icdiff

```

It looks like at one point, you probably had an 'icdiff' directory with an `__init__.py`. With the file structure as it currently is, the 'icdiff' script is not recognized as a Python module since it doesn't end with `.py` and so can't be imported.

**Comment by vnitinv** [comment]
You can just do "pip install icdiff"

$ pip install icdiff
Collecting icdiff
  Downloading icdiff-1.8.6.tar.gz
Installing collected packages: icdiff
  Running setup.py install for icdiff ... done
Successfully installed icdiff-1.8.6

**Comment by rcfox** [comment]
Thanks @vnitinv. It seems this repository is older than what is on the PyPI...

**Comment by oz123** [comment]
Since this project is already using entry points, it might make sense to simply rename `icdiff` to `icdiff.py`. This will solve the import error.

**Comment by jeffkaufman** [comment]
Fixed with version 1.9.0 by just symlinking `icdiff` to `icdiff.py`.  Now people can either install with standard setuptools and get an entry-point version, or they can just put `icdiff` on their `PATH`.

I've updated the project homepage to suggest `pip install git+https://github.com/jeffkaufman/icdiff.git` but once PyPI has 1.9.0 (@vnitnv ?) I can change that to just `pip install icdiff`.

**Comment by vnitinv** [comment]
@jeffkaufman Let me get this released over this weekend.
